contracts/callback-capturer/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "callback-capturer"
+version = "0.1.0"
+authors = ["Ethan Frey <ethanfrey@users.noreply.github.com>"]
+edition = "2021"
+publish = false
+license = "Apache-2.0"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+backtraces = ["cosmwasm-std/backtraces"]
+# use library feature to disable all instantiate/execute/query exports
+library = []
+
+[dependencies]
+simple-ica-controller = { version = "0.1.0", path = "../simple-ica-controller", features = ["library"] }
+simple-ica = { version = "0.1.0", path = "../../packages/simple-ica" }
+cw-utils = { version = "0.14.0" }
+cw2 = { version = "0.14.0" }
+cw-storage-plus = { version = "0.14.0" }
+cosmwasm-schema = { version = "1.0.0" }
+cosmwasm-std = { version = "1.0.0", features = ["staking", "stargate"] }
+schemars = "0.8.1"
+serde = { version = "1.0.103", default-features = false, features = ["derive"] }
+thiserror = { version = "1.0.23" }
+
+[dev-dependencies]
+anyhow = "1"
```

Comment:
Do we use anyhow here?

Reply:
Right, I think it was for the multi-test stuff I deleted after copying